sklearn/decomposition/tests/test_kernel_pca.py

```diff
@@ -108,31 +135,45 @@ def test_kernel_pca_sparse():
             assert (X_pred_transformed.shape[1] ==
                          X_fit_transformed.shape[1])
 
-            # inverse transform
-            # X_pred2 = kpca.inverse_transform(X_pred_transformed)
-            # assert X_pred2.shape == X_pred.shape)
+            # inverse transform: not available for sparse matrices
+            with pytest.raises(NotFittedError):
+                kpca.inverse_transform(X_pred_transformed)
```

Comment:
This exception type is surprising and therefore probably not appropriate. But since fixing this is not the main focus of this PR, we can just add a comment about this issue and delay the actual fix for a later PR.

```suggestion
            # inverse transform: not available for sparse matrices
            # XXX: should we raise another exception type here? For instance:
            # NotImplementedError.
            with pytest.raises(NotFittedError):
                kpca.inverse_transform(X_pred_transformed)
```